index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Document</title>
+    <link rel="stylesheet" href="style/index.css">
+</head>
+
+<body id="body">
+    <header>
+        <span class="light" id="light">&#9728;</span>
+        <span class="on-top"><a href="#body">&#8657;</a></span>
+    </header>
+    <main class="main-section">
+        <article class="first-information">
+            <img class="img-avatar" src="img/avatar.jpg" alt="avatar" height="300">
+            <div class="first-information_about">
+                <h1 class="name">Lazarevich Olga Igorevna</h1>
+                <h2>Contacts :</h2>
+                <ul>
+                    <li><span>Phone:</span> + 80257560060</li>
+                    <li><span>Github</span>: https://github.com/OlgaLazarevich</li>
+                    <li><span>Email:</span> helgalazarevich@gmail.com</li>
+                </ul>
+            </div>
+        </article>
+        <section class="section">
```

Comment:
Please, avoid using keywords as class names. It`s error prone and quite confusing to see the `<section>` element with the class **section**.
The rule of thumb: Don't use the language keywords(html, css, js) as the names for your variables.